### DISCUSSION TITLE
Which other sensor informations are possible?

### DISCUSSION THREAD

**Discussion by Skirf** [Jarnsen/ha-xsense-component_test#9] [Ideas]
Hi there,

thanks a lot for this integration! 

Now to have it work well, I need to build a Notification when an alarm goes off.

I use water, temperature und "normal" smoke sensors. Which informations do they provide which are not integrated atm? Maybe a temperature not only the alarm for temperature sensors?

Keep up the good work!

**Comment by Jarnsen** [comment]
In principle, everything that is stored in the app is possible. Theo is working step by step to implement everything. The temperature sensors do indeed transmit the temperature, as can be seen in the forum – Theo has posted a picture there. Triggering the alarm already works with an automation: the entity and status trigger when it switches to "On."

**Comment by Jarnsen** [comment]
at this time, i create a video to explain how it work to do an automation for this.
